docs/pages/enroll-resources/auto-discovery/auto-discovery.mdx

```diff
@@ -1,28 +1,124 @@
 
-The Discovery Service runs on [Teleport Agents](../agents/agents.mdx). It
-periodically queries cloud provider APIs to list resources in your
-infrastructure. It then reconciles these resources with Teleport resources
-registered on the Auth Service backend.
+import autoDiscoveryImg from "@version/docs/img/auto-discovery/auto-discovery-hero.jpg";
+import databaseAccessImg from "@site/src/components/Icon/teleport-svg/database-access.svg";
+import kubernetesClustersSvg from "@site/src/components/Icon/teleport-svg/kubernetes-clusters.svg";
+import linuxServersSvg from "@site/src/components/Icon/teleport-svg/linux-servers.svg";
+import kubernetesSvg from "@site/src/components/Icon/svg/kubernetes2.svg";
 
-Set up Teleport auto-discovery for resources in your infrastructure:
+<DocHero
+  title="Automatically discover and connect to resources"
+  image={autoDiscoveryImg}
+>
+Streamline infrastructure security with the Teleport Discovery Service. When you deploy servers, databases, and Kubernetes clusters, the service enables secure access to them with no further configuration needed.
 
-## Guides
+[Teleport Agents](../agents/agents.mdx) power the Discovery Service, automatically syncing resources with the Auth Service.
+</DocHero>
 
-- [Database Discovery (section)](./databases/): Detailed guides for configuring database discovery.
-- [Enroll Kubernetes Services as Teleport Applications (section)](./kubernetes-applications/): Teleport can automatically detect applications running in your Kubernetes clusters and register them with Teleport for secure access.
-- [Kubernetes Clusters Discovery (section)](./kubernetes/): Detailed guides for configuring Kubernetes Clusters Discovery.
-- [Server Auto-Discovery (section)](./servers/): You can set up the Teleport Discovery Service to automatically enroll servers in your infrastructure.
+<EnrollmentMethods title="Auto-discovery guides">
+<Method
+title="Databases"
+icon={databaseAccessImg}
+href="./databases/"
+tagLists={
+  [
+    {
+      tags: 
+        [
+          {
+            name: "AWS",
+            href: "./databases/aws/",
+            icon: "aws",
+          },
+          {
+            name: "Azure",
+            href: "../database-access/enroll-azure-databases/",
+            icon: "azure",
+          },
+        ] 
+    },
+  ]
+}
+>
+  Automatically discover and register databases in your cluster.
+</Method>
+<Method
+title="Servers"
+icon={linuxServersSvg}
+href="./servers/"
+tagLists={
+  [
+    {
+      tags: 
+        [
+          {
+            name: "Amazon EC2",
+            href: "./servers/ec2-discovery/",
+            icon: "ec2",
+          },
+          {
+            name: "Google Compute Engine",
+            href: "./servers/gcp-discovery/",
+            icon: "gSuite",
+          },
+         {
+            name: "Azure Virtual Machines",
+            href: "./servers/azure-discovery/",
+            icon: "entraId",
```

Comment:
Do we have a more generic Azure icon? You can use Azure VMs without using Entra ID.